pkg/virt-handler/non-root.go

```diff
@@ -132,7 +135,7 @@ func getTapDevices(vmi *v1.VirtualMachineInstance, networkBindings map[string]v1
 }
 
 func (d *VirtualMachineController) prepareTap(vmi *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
-	networkToTapDeviceNames, err := getTapDevices(vmi, d.clusterConfig.GetNetworkBindings())
+	networkToTapDeviceNames, err := getTapDevices(d.clusterConfig, vmi, d.clusterConfig.GetNetworkBindings())
```

Comment:
It would have been cleaner to pass in the network namescheme as input so the condition (about the FG) will occur at this level.

Reply:
Don't know, this works.